srcCpp/Infrastructure_pro.cxx

```diff
@@ -501,21 +501,118 @@ bool configureShmemTransport(
         DDS_DomainParticipantQos& qos,
         ParameterManager *_PM)
 {
-    // Number of messages that can be buffered in the receive queue.
-    int received_message_count_max = 1024 * 1024 * 2
-            / (int) _PM->get<unsigned long long>("dataLen");
-    if (received_message_count_max < 1) {
-        received_message_count_max = 1;
-    }
+    DDS_Property_t *parentProp = DDSPropertyQosPolicyHelper::lookup_property(qos.property,
+                "dds.transport.shmem.builtin.parent.message_size_max");
+    int parent_msg_size_max = atoi(parentProp->value);
```

Comment:
Follow naming convention:
`parent_msg_size_max` -> `parentMsgSizeMax`

Reply:
Apply also for the rest of the names.

Reply:
Done